presto-main/src/main/resources/webapp/tableau/presto-client.js

```diff
@@ -72,7 +74,7 @@ StatementClient.prototype.advance = function(lastRecordNumber) {
     var statementClient = this;
     $.ajax({
         type: "GET",
-        url: this.currentResults.nextUri,
+        url: this.isHttps ? this.currentResults.nextUri.replace("http", "https") : this.currentResults.nextUri,
```

Comment:
If `nextUri` already contains `'https'`, we might be replacing the prefix and end up with `'httpss'`. To be safer, I think we could match more of the string. We could also do the replacement the other way around (ie. switch from '`https'` to `'http'`) as well if `this.isHttps` is false.

Probably not necessary, but matching the prefix instead of first occurrence may add more safety (ie. `.replace(/^http/, 'https')`).